src/components/datatrust/FileUploader.vue

```diff
@@ -30,14 +30,16 @@ import UploadModule from '../../modules/UploadModule'
 import ListModule from '../../modules/ListModule'
 import { ProcessStatus } from '../../models/ProcessStatus'
 import { FileDropped } from '../../models/Events'
-import Dropzone from 'dropzone'
+import ComputableDropzone from 'dropzone'
 import { DropzoneFile } from 'dropzone'
 import uuid4 from 'uuid/v4'
+import SparkMD5, { hashBinary } from 'spark-md5'
+import Paths from '../../util/Paths'
 
 import '@/assets/style/components/file-uploader.sass'
 
-Dropzone.prototype.defaultOptions.dictDefaultMessage = 'Drag a file'
-Dropzone.autoDiscover = false
+ComputableDropzone.prototype.defaultOptions.dictDefaultMessage = 'Drag a file'
```

Comment:
why append to the prototype? are you instantiating more than one? why not just put it on after in those args you pass to the constructor?

If its cuz their shitty class isn't setup to let you override the `dictDefaultMessage` then its fine...

Reply:
it was early experimental code.  lemme look at passing it into `options`...